packages/components/package.json

```diff
@@ -127,26 +125,22 @@
         }
       ],
       [
-        "@semantic-release/exec",
+        "@semantic-release/npm",
         {
-          "prepareCmd": "pnpm build"
+          "npmPublish": false
         }
       ],
       [
-        "@semantic-release/npm",
+        "@semantic-release/exec",
         {
-          "npmPublish": false
+          "prepareCmd": "pnpm build && git add ../react && git commit -m \"chore: Create new version of @synergy-design-system/react [skip actions]\""
```

Comment:
Brauchen wir hier nicht noch einen git Push? Oder kommt der Push dann durch `@semantic-release/git`.

Irgendwie würde ich es eher einbauen, dass das save ist, oder?

Reply:
Just changed this in the latest version